website/pages/api-docs/volumes.mdx

```diff
@@ -323,6 +323,12 @@ The table below shows this endpoint's support for
   running allocations. This does not detach the volume from client nodes.
   This is specified as a query string parameter.
 
+- `detach` `(bool: false)`- Detach the volume from the node given by the `node`
```

Comment:
I'm really not wild about this approach, but volume HTTP API is fairly "restful" so I'm not really sure what would be a better way to go here. Maybe @DingoEatingFuzz has an opinion here?

Reply:
`/v1/volume/csi/:volume_id/detach` doesn't have the collision problem

Reply:
My imagination was failing me there.  🤦 

Do we like that better? I think it feels less gross.

Reply:
I like it a little better, but I don't know that it matters a lot.

The only technical consideration I can think of is I've seen proxies discard a DELETE body, but a) probably no proxy and b) query params may be fine. I've found some strong opinions strongly held that it's bad, but not much evidence that it's actually a problem 😁

Reply:
Per our internal discussion we're going to change this to be `/v1/volume/csi/:volume_id/detach`

Reply:
Done in 2e3ad64